lib/exception_handling/methods.rb

```diff
@@ -16,7 +16,9 @@ def log_error_rack(exception_or_string, exception_context = '', rack_filter = ''
     end
 
     def log_warning(message)
-      log_error(Warning.new(message))
+      warning = Warning.new(message)
+      warning.set_backtrace([])
+      log_error(warning)
```

Comment:
to avoid the duplication can this call `ExceptionHandling.log_warning(message)` to be more like the other methods in this module?